config/forms/2022_2023.json

```diff
@@ -5782,19 +6327,19 @@
               "depends_on": [
                 {
                   "hb": 1,
-                  "hbrentshortfall": 0
+                  "hbrentshortfall": 1
```

Comment:
Why is this changing?

Reply:
Ah looks like this is being fixed to match 21/22. :+1:

Reply:
Might need to check whether this affects any of the logs already submitted since they're potentially missing a field.